### PR TITLE
Feature/nm 9

### PR DESCRIPTION
**Pull request by katarinakardosova01**

### CODE REVIEW

File: app.js.orig [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+app.get('/user-active-days', (req, res, next) => activeDays.getActiveDaysForUsername(req, res, next));
+app.get('/team-active-days', (req, res, next) => activeDays.getActiveDaysForTeam(req, res, next));
+app.get('/teams-scores', (req, res, next) => score.getTeamsScores(req, res, next));
+
+// Testing endpoints
+app.get('/userStats', (req, res, next) => github.getGithubDataForUsername(req, res, next));
+app.get('/teamStats', (req, res, next) => github.getGithubDataForTeam(req, res, next));
+app.get('/contribution', (req, res, next) => github.getContribution(req, res, next));
+
+<<<<<<< HEAD
```
Comment (SilviaKorcekova) [inline app.js.orig]: please resolve this

File: controllers/commits.controller.js [hunk 2]
```diff
@@ -25,22 +25,22 @@ function getCommitsNumberForUser(req, res, next){
...
     if(!teamName) {
-        throw new Error(`Missing required parameter: teamName. `)
+        throw new Error(`Missing required parameter: team name. `)
     }
 	
     let teamRepos = utilsService.getRepositoriesForTeam(teamName);
     let teamMembers = utilsService.getTeamMembers(teamName);
 
     let commits = commitsService.getCommitForTeam(teamRepos, teamMembers, startDate, endDate)
         .then(commits => res.json({
```
Comment (ajancus) [inline controllers/commits.controller.js]: This logic should be moved to service

File: controllers/commits.controller.js [hunk 3]
```diff
@@ -25,28 +25,33 @@ function getCommitsNumberForUser(req, res, next){
...
-        .then(commits => res.json({
-                teamName : teamName,
-                commitsCount : commits.length,
-                startDate : startDate,
-                endDate : endDate
-            }
-        ));
+    if(!teamRepos) {
+        throw new Error(`No team exists with name: ` + teamName);
+    } else {
```
Comment (ajancus) [inline controllers/commits.controller.js]: Can you delete this unneccessary else?

File: controllers/commits.controller.js [hunk 4]
```diff
@@ -25,28 +25,33 @@ function getCommitsNumberForUser(req, res, next){
...
-    let commits = commitsService.getCommitForTeam(teamRepos, teamMembers, startDate, endDate)
-        .then(commits => res.json({
-                teamName : teamName,
-                commitsCount : commits.length,
-                startDate : startDate,
-                endDate : endDate
-            }
-        ));
+    if(!teamRepos) {
+        throw new Error(`No team exists with name: ` + teamName);
```
Comment (ajancus) [inline controllers/commits.controller.js]: No githubRepositories found for team?

**Pull request merged**